gazebo_msgs/msg/InstantSlip.msg

```diff
@@ -0,0 +1,3 @@
+string[] name
```

Comment:
can you add documentation to this message file? you can use the joint states message as a reference:

* https://github.com/ros2/common_interfaces/blob/master/sensor_msgs/msg/JointState.msg

Reply:
Fixed in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1331/commits/74450d05243da4f06905d7551590e167e905c534.